### PR TITLE
Added some missing types from various stdlib stubs

### PR DESCRIPTION
**Pull request by erictraut**

### CODE REVIEW

File: stdlib/2and3/socket.pyi [hunk 1]
```diff
@@ -713,11 +713,11 @@ class socket:
             self, __buffers: Iterable[bytes], __ancdata: Iterable[_CMSG] = ..., __flags: int = ..., __address: _Address = ...
         ) -> int: ...
     if sys.platform == "linux" and sys.version_info >= (3, 6):
-        # TODO add the parameter types for sendmsg_afalg
-        def sendmsg_afalg(self, msg=..., *, op, iv=..., assoclen=..., flags=...) -> int: ...
+        def sendmsg_afalg(
+            self, msg: Sequence[bytes] = ..., *, op: int, iv: Any = ..., assoclen: int = ..., flags: int = ...
```
Comment (srittau) [inline stdlib/2and3/socket.pyi:717]: Is `Sequence[bytes]` really the correct type? I am not familiar with this function, but this looks suspicious to me.

File: stdlib/3/multiprocessing/dummy/__init__.pyi [hunk 2]
```diff
@@ -19,7 +19,14 @@ class DummyProcess(threading.Thread):
     _pid: None
     _start_called: int
     exitcode: Optional[int]
-    def __init__(self, group=..., target=..., name=..., args=..., kwargs=...) -> None: ...
+    def __init__(
+        self,
+        group: None = ...,
```
Comment (srittau) [inline stdlib/3/multiprocessing/dummy/__init__.pyi:24]: I think it's better to use `Any` or whatever other process classes accept here. The argument is ignored for `DummyProcess`, but I surmise it exist for compatibility with other process classes.

File: stdlib/3/asyncio/trsock.pyi [hunk 3]
```diff
@@ -44,13 +44,17 @@ if sys.version_info >= (3, 8):
...
         if sys.platform == "linux":
-            def sendmsg_afalg(self, msg=..., *, op, iv=..., assoclen=..., flags=...) -> int: ...
+            def sendmsg_afalg(
+                self, msg: Sequence[bytes] = ..., *, op: int, iv: Any = ..., assoclen: int = ..., flags: int = ...
+            ) -> int: ...
         else:
-            def sendmsg_afalg(self, msg=..., *, op, iv=..., assoclen=..., flags=...) -> NoReturn: ...
+            def sendmsg_afalg(
+                self, msg: Sequence[bytes] = ..., *, op: int, iv: Any = ..., assoclen: int = ..., flags: int = ...
+            ) -> NoReturn: ...
```
Comment (srittau) [inline stdlib/3/asyncio/trsock.pyi:57]: See above.

**Review by srittau — APPROVED** [review]
I am not familiar with most of the modules changed, but a cursory examination looks good to me, except for two potential issues.

File: stdlib/2and3/socket.pyi (hunk 1 above)
Comment (JelleZijlstra) [reply stdlib/2and3/socket.pyi:717]: From looking at the C code it seems like it should be `Iterable[bytes]`.

Comment (erictraut) [reply stdlib/2and3/socket.pyi]: I made the recommended change.

File: stdlib/3/asyncio/trsock.pyi (hunk 3 above)
Comment (erictraut) [reply stdlib/3/asyncio/trsock.pyi:57]: I made the recommended change.

File: stdlib/3/multiprocessing/dummy/__init__.pyi (hunk 2 above)
Comment (erictraut) [reply stdlib/3/multiprocessing/dummy/__init__.pyi]: I made the recommended change.

**Pull request merged**